source/whats-new.txt

```diff
@@ -37,6 +38,19 @@ Learn what's new in:
 .. In addition to the deprecations mentioned in specific driver versions on this
    page, anticipated breaking changes include the following:
 
+.. _version-5.0:
+
+What's New in 5.0
+-----------------
+
+New features of the 5.0 driver release include:
```

Comment:
S: since the style guide recommends introducing a list with a full sentence, maybe reword to something like:

```suggestion
The 5.0 driver introduces the following features:
```